### PR TITLE
Submit via enter

### PR DESCRIPTION
**Pull request by frederike-ramin**
I actually implemented this during hour hack hours, but forgot to push. So, now you can submit by pressing Enter.

### CODE REVIEW

File: tests/test.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ class TimeEntriesPage {
     await this.page.locator('label:has-text("to:")').fill(to);
   }
 
-  async addEntry(year, month, date, hours, description) {
+  async addEntry(year, month, date, hours, description, useEnter = false) {
```
Comment (mpanne) [inline tests/test.js:27]: I think you added the parameter to the wrong method head. Move this to the "editEntry" method.

**Review by mpanne — CHANGES_REQUESTED** [review]

File: tests/test.js (hunk 1 above)
Comment (frederike-ramin) [reply tests/test.js:27]: You are absolutely correct. Merge conflict 🚨

**Review by mpanne — CHANGES_REQUESTED** [review]
This feature causes problems with the edit function. If you edit an entry and then hit the "Enter" key, the "submit" function is executed and the entry is added but not deleted (since the edit function is not executed).

**Comment by frederike-ramin** [comment]
Alrighty, thanks for taking such a close look @mpanne, I'm gonna close this for now because fixing the issue would require more than 10min of my time, so maybe let's revisit this some other time. :)

**Comment by frederike-ramin** [comment]
So simple and so great. 👌 
You wanna add a test maybe? If not, fine with me^^

Fun Fact: I can't approve this as it is my PR - consider it approved though. :D

**Review by mpanne — APPROVED** [review]
Added some tests.

**Pull request merged**